lib/nav_tab/link.rb

```diff
@@ -0,0 +1,30 @@
+class NavTab::Link
+  include ActionView::Helpers::UrlHelper
+
+  attr_reader :subnav, :text, :url
+
+  def initialize(tab: nil, text: nil, url: nil, subnav: nil, cross_facility: false)
+    @tab = tab.present? && tab.to_s
```

Comment:
I think `tab.presence` will give you the same thing, nil if empty/blank and the value otherwise.